content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,230 @@ aliases:
+Istio also supports authentication in permissive mode to understand how a policy change would affect your security posture
+before it becomes effective.
 
-    - Replace `_domain_` with `_cluster.local_`. We will support customization
-      of domain in the near future.
-    - Replace `_namespace_` with the namespace of the Kubernetes service
-      account.
-    - Replace `_serviceaccount_` with the Kubernetes service account name.
+For more details,  please read the [authentication](/docs/concepts/security/#authentication) section.
 
-- A service account is **the identity or role a workload runs as**. The service
-  account represents that workload's privileges. For systems requiring strong
-  security, neither a random string, such as a service name, label, etc., nor
-  the deployed binary should identify the amount of privilege for a workload.
-  For example, let's say we have a workload pulling data from a multi-tenant
-  database. If Alice runs this workload, she pulls a different set of data than
-  if Bob runs this workload.
+### Authorization & audit
 
-- To enable strong security policies, service accounts offer the flexibility
-  to identify a machine, a user, a workload, or a group of workloads. Different
-  workloads can even run as the same service account.
+Istio provides fine-grained access control to enable application-level control and auditing on who accesses your service, API, or resource,
+using a variety of access control mechanisms.
+They include attribute and role-based access control as well as authorization hooks.
 
-- The service account a workload runs as won't change during the lifetime of
-  the workload.
+Find out more about Istio authorization in the [authorization guide](/docs/concepts/security/#authorization).
+The guide for auditing is coming soon!
 
-- With domain name constraint, you can ensure service account uniqueness.
+### Security configuration
 
-### Communication security
+Istio makes it simple to configure and enable security features using Istio's rich yet easy-to-use policy creation
+and centralized policy management.
 
-Istio tunnels service-to-service communication through the client side
-[Envoy](https://envoyproxy.github.io/envoy/) and the server side Envoy.
-Istio secures end-to-end communication via:
+Find out more about configuring security policies in the [authentication](/docs/concepts/security/#authentication)
+and [authorization](/docs/concepts/security/#authorization) sections.
 
-- **Local TCP** connections between the service and Envoy.
+### Using other security mechanisms
 
-- **Mutual TLS** connections between proxies.
+While we strongly recommend using Istio security,
+Istio is flexible enough to allow you to plug in your own authentication and authorization mechanisms via the Mixer component.
+You can find out how to use and configure plugins in [Mixer policies and telemetry](/docs/concepts/policies-and-telemetry/#adapters).
```

Comment:
To simplify and add clarity:
To use and configure plugins in Mixer, visit our [policies and telemetry adapters docs](/docs/concepts/policies-and-telemetry/#adapters).